src/pivotal-ui/components/header/header.hbs

```diff
@@ -0,0 +1,117 @@
+<header class="{{#if user}}logged-in{{else}}anonymous{{/if}} {{#if headerLight}}header-light{{/if}} {{#if heroTransparent}}hero-transparent{{/if}} {{#if features.npmo}}header-enterprise{{/if}}">
+  <!-- logo -->
+  <div class="header-item brand-container">
+    <div class="logo-container" id='npm-logo'>
+      <a href="/">{{svg "@npmcorp/pui-css-header/npm-logo.svg"}}</a>
+      {{#if env.NPMO_COBRAND}}
+        <span class="cobrand">{{env.NPMO_COBRAND}}</span>
+      {{/if}}
+    </div>
+  </div>
+  <!-- /logo -->
+
+  <!-- search -->
+  <div class="header-item header-search-container">
+    <form class="site-search" id="npm-search" action="/search" method="get" itemprop="potentialAction" itemscope itemtype="http://schema.org/SearchAction">
+      <meta itemprop="target" content="{{env.CANONICAL_HOST}}/search?q={q}">
+      <div id="site-search-container">
+        {{#if features.npmo}}
+          <input name="q" value="{{q}}" type="search" id="site-search" placeholder="find on-site packages" tabindex="1" autocorrect="off" autocapitalize="off" itemprop="query-input" />
+        {{else}}
+          <input name="q" value="{{q}}" type="search" id="site-search" placeholder="find packages" tabindex="1" autocorrect="off" autocapitalize="off" itemprop="query-input" />
+        {{/if}}
+      </div>
+      <button type="submit">
+        <span class="a11y-only">Search</span>
+        {{svg "@npmcorp/pui-css-header/ico-search.svg"}}
+      </button>
+    </form>
+  </div>
+
+  {{#if features.npmo}}
```

Comment:
I'm not invested either way. I'd rather move toward unifying them, but that'll of course depend on designs.